exchanges/bitstamp/bitstamp_types.go

```diff
@@ -160,34 +158,49 @@ const (
 	errStr                  string = "error"
 )
 
-// WebsocketConn defines a pusher websocket connection
-type WebsocketConn struct {
-	Client *pusher.Client
-	Data   chan *pusher.Event
-	Trade  chan *pusher.Event
+// WebsocketEventRequest contains event data for a websocket channel
+type websocketEventRequest struct {
+	Event string        `json:"event"`
+	Data  websocketData `json:"data"`
+}
+
+type websocketData struct {
+	Channel string `json:"channel"`
+}
+
+type websocketResponse struct {
+	Event   string `json:"event"`
+	Channel string `json:"channel"`
+}
+
+type websocketTradeResponse struct {
+	Data    websocketTradeData `json:"data"`
+	Event   string             `json:"event"`
```

Comment:
Can put websocketResponse in here as an anonymous struct as it uses the same fields